src/content/structured/get-started/install-components.mdx

```diff
@@ -56,3 +56,7 @@ To use the components in a particular framework, follow the framework instructio
   </ic-link>
   .
 </p>
+
+## Using 3rd party libraries alongside ICDS
```

Comment:
Just wondering if we should be using "third" instead of "3rd"? We have a [section about numbers](https://design.sis.gov.uk/styles/content-style/numbers-dates-and-time) which says to use the word for small numbers, so thought it might be a similar case for "3rd", "2nd" etc.

Reply:
@DBD324, what are your thoughts on this? Is '3rd' acceptable or does this information page on numbers contradict it?

Reply:
Was just about to suggest the same thing. Our content style indicates that we should use words for numbers below 10 and numbers for those over 10. 

Lets go for using 'third-party' and make sure we are consistent with the hyphenation too.